doc/api/v1.yaml

```diff
@@ -36,15 +36,55 @@ paths:
           content:
             application/json:
               schema:
-                description: A list of fragment IDs (same order as in input)
-                type: array
-                items:
-                  type: string
-                  pattern: '[0-9a-f]+'
-              example:
-                [
-                  '68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968',
-                ]
+                description: The information about whether a message was accepted or rejected
```

Comment:
These are big and should probably be reused...

Reply:
Dunno, we should probably migrate from v0 to v1-only API at some point. But maybe type definitions can be split into a separate file.